mesheryctl/pkg/utils/helpers.go

```diff
@@ -86,6 +86,7 @@ var TemplateContext = models.Context{
 	},
 	Platform: "docker",
 	Adapters: ListOfAdapters,
+	Channel:  "",
```

Comment:
```suggestion
	Channel:  "stable",
```

`stable` will be the default channel.